crypto/crypto.c

```diff
@@ -153,4 +153,8 @@ int OPENSSL_init_crypto(uint64_t opts, const OPENSSL_INIT_SETTINGS *settings) {
   return 1;
 }
 
+void OPENSSL_init(void) {
+    return;
+}
```

Comment:
NP pattern seems to be empty, I could see some compilers getting tripped up on this. 
```suggestion
void OPENSSL_init(void) {}
```